week5.1-ex1.py

```diff
@@ -0,0 +1,18 @@
+import os
+
+
+def files_start_with(directory_path):
```

Comment:
Read about type hints and return types -https://docs.python.org/3/library/typing.html
Your definition should look like this - 
![Capture](https://user-images.githubusercontent.com/13467749/159873588-f7ab7768-9dbc-4c8e-86bb-9f5b5a3748d2.JPG)